src/ig/IGExporter.ts

```diff
@@ -415,17 +419,130 @@ export class IGExporter {
 
   /**
    * Adds any user provided includes files
-   * A user provided menu.xml will be in this folder. If one is not provided, the static one SUSHI provides will be used.
+   * A user provided menu.xml may be in this folder, but do not handle it here. It is handled separately in addMenuXML.
    *
    * @param {string} igPath - the path where the IG is exported to
    */
   private addIncludeContents(igPath: string): void {
     const includesPath = path.join(this.igDataPath, 'input', 'includes');
     if (existsSync(includesPath)) {
-      this.copyWithWarningText(includesPath, path.join(igPath, 'input', 'includes'));
+      this.copyWithWarningText(includesPath, path.join(igPath, 'input', 'includes'), src => {
+        // Filter out menu.xml because handled separately
+        if (path.parse(src).base.startsWith('menu.xml')) return false;
+        return true;
```

Comment:
Ah that's a good point. I was mirroring the other place where we have a filtering function in the call to `copyAsIs` in `addStaticFiles`, but I can change it here since this filter is much simpler than that one.